### ISSUE TITLE
Incorrect Indication of 6-Pos Switch State through the LEDs

### ISSUE THREAD

**Issue by Hartinger** [FrSkyRC/ETHOS-Feedback-Community#4155]
I use a Tandem X20S with ETHOS 1.5.9. I found that on power-up, or after changing the model, the Blue LEDs of the 6-Pos Switch do not indicate the correct state of the switch. Only after any of the 6 buttons is pressed, the LED display reflects the state of the switch.

Two examples:

(1) When the switch configuration is set to 6-Pos (Default), then on power-up no LED is lit at all, hence the operating state of the switch is not visible (by definition always one LED must be on for this configuration). Only when a button is pressed, then the corresponding LED is lit. From then any button press correctly lits the LED associated with the selected button.

(2) The switch is configured as 6 individual ON/OFF switches for a particula model, and some of the buttons are pressed. The corresponding LEDs correctly indicate which switches are ON. However, when I then select a different model, the LED status remains unchanged, even if for the new model the switch is configured as a 6-Pos switch, where per definition only one LED can be on.

I could add more examples, it seems that on power-up or model change, the LED status is not updated to reflect the configuration of the switch and its operating state.

**Comment by Wolepo** [comment]
To 1. is the first or the 2nd line set?
![FS-Einstellungen](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/90325526/2e2a0909-0316-4223-a12b-360453aea8f5)
If it is set to "6-Pos with OFF", then it would be logical if no LED lights up.

**Comment by Hartinger** [comment]
No, it is set to the default, which is 6-Pos (no OFF state).

**Comment by bsongis-frsky** [comment]
I think it is duplicate of #4161 
Would you please have a test on the next nightlies?

**Comment by Hartinger** [comment]
I have tested with the latest nighlty.

Problem (2) in my initial post has been cured. If the switch is configured as 6 individual ON/OFF switches for a particula model, and some of the buttons are pressed, the corresponding LEDs go out upon selection of a different model.

Regarding (1) in my initial post, there is no change in behaviour. Having read the manual, this seems to be working as designed, i.e. all switches are initially OFF, regardless how the switch is configured.
However, if the switch is configured as a 6-Pos switch, should then not always be exactly one switch ON, including upon selection of the model? Otherwise, the 6-Pos option is actually redundant, as the 6-Pos with OFF option provides the same functionality. And with the 6-Pos with OFF option you can restore the initial stae (all switches OFF), by repressing the selected switch. With the 6-Pos option, once you have pressed any button, there is no possibility to restore the initial state, which does not feel right.
